linkcheck/checker/mailtourl.py

```diff
@@ -57,8 +57,8 @@ def is_literal (domain):
     return domain.startswith(u'[') and domain.endswith(u']')
 
 
-_remove_quoted = re.compile(ur'\\.').sub
-_quotes = re.compile(ur'["\\]')
+_remove_quoted = re.compile(r'\\.').sub
+_quotes = re.compile(r'["\\]')
```

Comment:
Like mentioned on #212, I'm not entirely sure this is safe on Python 2.

Reply:
Made `u` as #212 with doubled `\`.

Reply:
Reverted back to:
```py
_remove_quoted = re.compile(r'\\.').sub
_quotes = re.compile(r'["\\]')
```